tests/dummy/app/pods/demo/error/template.hbs

```diff
@@ -44,10 +44,10 @@
     isVisible=false
   )
   hook='error-dialog'
-  footer=(component 'frost-loading' class='frost-footer-component')
+  footer=(component 'frost-loading' hook='frost-loading' class='frost-footer-component')
```

Comment:
What is the need for this change?  There are no corresponding changes to any tests.

Reply:
demo page breaks because of missing `hook` property

Reply:
👍